src/components/StarRating.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
 function StarRating(props) {
-  const { product } = props;
-  const arr = Array(product.rating).fill(null);
+  const { rating } = props;
```

Comment:
Could also just use a normal for loop. This was to show that it isn't required to use map all the time